frontend/common/src/components/CheckButton/CheckButton.stories.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { Story, Meta } from "@storybook/react";
+import { action } from "@storybook/addon-actions";
+import CheckButton from "./CheckButton";
+
+export default {
+  component: CheckButton,
+  title: "Components/Check Button",
+  args: {
+    label: "Check button label",
+  },
+} as Meta;
+
+const Template: Story<{ label: string }> = (args) => {
+  const [isChecked, setChecked] = React.useState<boolean>(false);
+  const { label } = args;
+
+  const handleCheck = () => {
+    const newState = !isChecked;
+    setChecked(newState);
+    action(`Changed state: ${newState}`);
+  };
+
+  return (
+    <CheckButton checked={isChecked} onToggle={handleCheck} label={label} />
+  );
+};
+
+export const BasicCheckButton = Template.bind({});
```

Comment:
Might be worth having a story that includes indeterminate state with the MinusIcon as well.

Reply:
Updated with [add more stories for check button](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3279/commits/23063720c7e9c607bd38c815c14038c2ffdc57a7)